source/source-connector/usage-examples/topic-naming.txt

```diff
@@ -199,19 +199,19 @@ any other purpose.
    When you create a namespace pattern, ensure that you properly handle
    characters that need escaping. For example, to match
    ``.``, the regex syntax requires that you escape it as ``\.``
-   However, you must escape the backslash ``\`` character as ``\\``.
-   Then, to match ``.`` in your namespace pattern, you must
-   write it as ``\\.``.
+   However, you must escape the backslash ``\`` character as ``\\``
+   according to the JSON syntax. Then, to match ``.`` in your namespace
+   pattern, you must write it as ``\\.``.
 
 This example shows how to specify the ``topic.namespace.map``
 setting so that the connector performs the following mapping:
 
 - Writes events from databases that match a regular expression to the
-  ``industrial{sep_coll}`` topic. The regex pattern matches any
+  ``industrial{sep_coll}`` topic name template. The regex pattern matches any
```

Comment:
Replacing "topic" with "topic name template" where appropriate without changing the surrounding text results weird text, for example:

> Writes events from databases that match a regular expression to the `industrial{sep_coll}` topic name template.

Events cannot be written to a topic name template, but they can be written to a topic that was determined based on the topic name template. I am pointing this out just in case you want to work on the wording, I don't see it as part of the technical review.